packages/worker/src/index.ts

```diff
@@ -89,8 +88,8 @@ const shutdown = () => {
 export default server.listen(parseInt(env.PORT || "4002"), async () => {
   console.log(`Worker running on ${JSON.stringify(server.address())}`)
   await initPro()
-  await redis.invite.init()
-  await redis.passwordReset.init()
+  await redis.clients.getInviteClient()
```

Comment:
This might be confusing. Could we not keep the init function, and to call the get clients inside? It's strange that on app limit we are getting the clients to then not use it.

Reply:
fair - I'll just expose the init